src/uniprotkb/components/entry/CommunityCuration.tsx

```diff
@@ -0,0 +1,111 @@
+import { useState } from 'react';
+import {
+  Button,
+  ExternalLink,
+  HeroContainer,
+  CommunityAnnotationIcon,
+  ChevronUpIcon,
+  ChevronDownIcon,
+} from 'franklin-sites';
+
+import {
+  Reference,
+  SourceCategory,
+} from '../../../supporting-data/citations/adapters/citationsConverter';
+
+import externalUrls from '../../../shared/config/externalUrls';
+
+import styles from './styles/community-curation.module.scss';
+
+import ORCIDiDLogo from '../../../images/ORCIDiD_icon.png';
+
+export const filterReferencesByCategory = (
+  communityReferences: Reference[],
+  category: SourceCategory
+) =>
+  communityReferences.filter((reference) =>
+    reference.sourceCategories?.includes(category)
+  );
+
+const CommunityCuration = ({
+  accession,
+  communityReferences,
+}: {
+  accession: string;
+  communityReferences: Reference[];
+}) => {
+  const [toggleView, setToggleView] = useState(false);
+  if (communityReferences.length) {
+    return (
+      <div>
+        <div className={styles.header}>
+          <Button
+            variant="tertiary"
+            className={styles['community-curation-button']}
+            onClick={() => setToggleView(!toggleView)}
+          >
+            <CommunityAnnotationIcon />
+            {`Community curation (${communityReferences.length}) `}
+            {toggleView ? (
+              <ChevronUpIcon width="1ch" />
+            ) : (
+              <ChevronDownIcon width="1ch" />
```

Comment:
The only place where you'd need a state would be here, for styling, so as a workaround as the open/closed state is on the html details element then you could just render one chevron, and transform it to flip it in CSS with the selector `[open]` for the parent details element